app/src/pages/common/settingsPage/settingsPage.jsx

```diff
@@ -136,14 +130,19 @@ export class SettingsTabs extends Component {
     return tabsConfig;
   };
 
-  render = () => (
-    <div className={cx('settings-tabs')}>
-      <NavigationTabs
-        config={this.createTabsConfig()}
-        activeTab={this.props.activeTab}
-        onChangeTab={this.props.changeTab}
-        mobileDisabled
-      />
-    </div>
-  );
+  render() {
+    return (
+      <PageLayout>
```

Comment:
In case of admin page, it will become nested into another page section which will result in additional padding.
I think it's better to leave tabs here, and extract page layout (with header) one level upper (to corresponding containers).

Reply:
Fixed